ebean-ddl-generator/src/main/java/io/ebeaninternal/dbmigration/ddlgeneration/platform/DB2Ddl.java

```diff
@@ -42,6 +46,16 @@ public DB2Ddl(DatabasePlatform platform) {
     this.inlineUniqueWhenNullable = false;
   }
 
+  @Override
+  public String alterTableTablespace(String tablename, String tableSpace, String indexSpace, String lobSpace) {
+    if(tableSpace == null) {
+      // if no tableSpace set, use the default tablespace USERSPACE1
```

Comment:
Default TS ist USERSPACE1: https://www.tutorialspoint.com/db2/db2_tablespaces.htm#:~:text=In%20a%20database%2C%20we%20have,tablespace%20contains%20temporary%20table%20data.